LEGO1/lego/legoomni/src/build/legocarbuild.cpp

```diff
@@ -559,7 +561,7 @@ MxResult LegoCarBuild::Tickle()
 	}
 
 	if (m_unk0x10a) {
-		DWORD time = timeGetTime();
+		Uint64 time = SDL_GetTicks();
 		DWORD dTime = (time - m_unk0x10c) / 100;
```

Comment:
I think it would be good if `DWORD` also becomes `Uint64` here.

Reply:
I hope the most significant bits are never used. If they are, then we have a big problem :)

```suggestion
		Uint64 dTime = (time - m_unk0x10c) / 100;
```